Gruntfile.js

```diff
@@ -410,6 +428,7 @@ module.exports = function (grunt) {
   grunt.registerTask('build:cr17', [
     'sass:cr17',
     'modernizr',
+    'imagemin:cr17',
```

Comment:
Mmmm do we need this update for this SR work?

Reply:
probably not, will remove for now